packages/dd-trace/src/util.js

```diff
@@ -15,7 +15,9 @@ function isError (value) {
     return true
   }
   if (value && value.constructor) {
-    return value.constructor.name === 'JestAssertionError' || value.constructor.name === 'Error'
+    return value.constructor.name === 'JestAssertionError' ||
+      value.constructor.name === 'Error' ||
+      value.constructor.name === 'ErrorWithStack'
```

Comment:
again, something weird with errors and `jest`: https://github.com/facebook/jest/blob/3b253f83b7500f2d5b627f4ab5f6971e7ef8349c/packages/jest-util/src/ErrorWithStack.ts#L8